### ISSUE TITLE
bug: Cannot create resource group if machine tags have not been seen by tenant.

### ISSUE THREAD

**Issue by getsec** [lacework/terraform-provider-lacework#263]
**Describe the bug**
`[400] No integration or data found for this Resource Type` when deploying `lacework_resource_group_machine`

**To Reproduce**

**Terraform:**
```hcl
resource "lacework_resource_group_machine" "test" {
  name        = "blah-test-machines"
  description = "test security machines"
  machine_tags {
    key   = "mode"
    value = "blah"
  }
}
```

**Command:** `terraform apply`

**Output:** 

```
[POST] https://menlosecurity.lacework.net/api/v2/ResourceGroups
[400] No integration or data found for this Resource Type

```

**Expected behaviour**
Since we're using lacework for tenant creation. We want to automatically add these resource groups to an account knowing these machine tags WILL be available even if they are not there today. Currently, I cannot add any resource groups unless the metadata for which I'm referring too already exists in that tenant.

```
Terraform v0.13.6
provider registry.terraform.io/lacework/lacework v0.15.0
```

**Comment by afiune** [comment]
Hello @getsec 👋🏽  - Thank you for your feedback. 

This error message indicates that your account (or sub-account if you have a Lacework organization) does not have an agent installed (reporting to your account). As soon as you have one agent reporting back to your account you can start creating Resource Groups for Machines.

Having said all that, I will pass along your feedback to our platform team for their consideration, maybe allowing users to create any Resource Group even if they are not using the feature could be a possible solution.

Please, try installing at least one agent and re-run your terraform code, let me know if you still have the same problem.

**Comment by afiune** [comment]
Jira: https://lacework.atlassian.net/browse/ALLY-866

**Comment by afiune** [comment]
Hi there 👋🏽  @getsec - Just checking in with you about this issue.

**Comment by afiune** [comment]
We have changed the logic in our backend and we are now allowing users to create Resource Groups
of types that the platform hasn't seen before such as containers, hosts, etc. We took this approach
since we understand that customers like you would like to preconfigure new accounts with specific
configurations even before provisioning things in the account itself. (order should not matter now)

Thank you so much for your feedback!
